src/odin/maneuversbuilder.cc

```diff
@@ -486,6 +524,26 @@ void ManeuversBuilder::Combine(std::list<Maneuver>& maneuvers) {
         LOG_TRACE("+++ Combine: obvious maneuver +++");
         next_man = CombineManeuvers(maneuvers, curr_man, next_man);
         maneuvers_have_been_combined = true;
+      }
```

Comment:
Suggestion: split these changes into different PR so the testing/validation will be easier